apps/Reaper/install-32

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+install_packages xz-utils tar || exit 1
+cd /tmp
+wget -O reaper.tar.xz "https://www.reaper.fm/files/6.x/reaper656_linux_armv7l.tar.xz" || error "Failed to download reaper!"
+status "Extracting reaper..."
+tar -xf reaper.tar.xz || error "Failed to extract reaper!"
+cd reaper_linux_armv7l || error "Failed to cd into 'reaper_linux_armv7l'"
+status_green "Done!"
+status "Installing reaper..."
+chmod +x install-reaper.sh || error "Failed to make 'install-reaper.sh' executable!"
+sudo ./install-reaper.sh --install /opt || error "Failed to run 'sudo install-reaper.sh --install /opt'"
+sudo rm -f /usr/local/bin/reaper &>/dev/null
+sudo ln -s /opt/REAPER/reaper /usr/local/bin/reaper || error "Failed to link /opt/REAPER/reaper to /usr/local/bin/reaper!"
+mkdir -p "$HOME/.local/share/icons/hicolor/64x64/apps/" || error "Failed to make icon directory!"
+cp "${DIRECTORY}/pi-apps/apps/Reaper/icon-64.png" "$HOME/.local/share/icons/hicolor/64x64/apps/reaper.png" || error "Failed to copy icon!"
```

Comment:
@theofficialgman isn't the `$DIRECTORY` variable equivalent to `/location/of/pi-apps`? If so, shouldn't this line be:
```suggestion
cp "${DIRECTORY}/apps/Reaper/icon-64.png" "$HOME/.local/share/icons/hicolor/64x64/apps/reaper.png" || error "Failed to copy icon!"



```

Reply:
oh yeah it is. my bad. good catch